core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -605,11 +588,30 @@ def confirm_blob(
         chunkify=chunkify,
     )
 
-    return raise_if_not_confirmed(
-        layout,
-        br_name,
-        br_code,
-    )
+    if ask_pagination and layout.page_count() > 1:
+        assert not hold
+        return _confirm_ask_pagination(br_name, title, data, description or "", br_code)
+    else:
+        return raise_if_not_confirmed(
+            layout,
+            br_name,
+            br_code,
+        )
+
+
+if not utils.BITCOIN_ONLY:
+
+    def confirm_other_data(data: bytes, data_total: int) -> Awaitable[None]:
```

Comment:
this is strongly tied to Ethereum and as such i'd say it should go into `apps/ethereum/layout.py` ?

Reply:
please try using "x" for confirm_blob _always_ on TT/TR and see if any use of `verb_cancel` actually pops up.

Reply:
I see what you mean. Basically ignoring the `verb_cancel` parameter, if possible, on these models.

The issue I see is that sometimes on TR verb_cancel is `"<"` or `"^"`. Arguably, this misused, since these are more like "verb_back" than "verb_cancel", but then we would end up complicating it even more - how do we know whether a layout has a back or a cancel?

Maybe I can make it such that, on model TR / TT `verb_cancel` can be 1 character long - otherwise it would become `"x"`?

Reply:
there's just one case that calls `ui.layouts.confirm_blob` with verb_cancel, and that's in impl `confirm_signverify`
where I think you could replace it with `trezorui2.confirm_blob` without loss of functionality

are there others that I missed?

Reply:
in a signverify ? just drop it

Reply:
https://github.com/trezor/trezor-firmware/pull/4343/commits/7abacb3ff7d16d9a9a06e6eca0dc957171e8936c